### DISCUSSION TITLE
Improving Awesome Scala landing page

### DISCUSSION THREAD

**Discussion by adpi2** [scalacenter/scaladex#1000] [Ideas]
@ichoran said in this [post on Scala Contributors](https://contributors.scala-lang.org/t/scala-center-q1-2022-update/5680/2)

> Awesome Scala is awesome, but it is a bit weird that you’re dropped directly into “Asynchronous, Concurrent, and Distributed Programming”. I mean…that stuff is awesome, of course. But it would be better if the landing page was in some sense the most awesome-of-the-awesome (by popularity) or picked out some great libraries at random, or something like that. Unless you have data to indicate that async is the most in-demand thing, this seems like alphabetization driving a pretty important first impression. Like, personally, I would think having sbt, mill, better-files, spark, etc., on the landing page would make more sense.

**Comment by adpi2** [comment]
The goal of this page is to give an overview of the Scala ecosystem in all its richness: diversity of use cases and platforms (JVM, Scala.js and Scala Native). It showcases the most used libraries, like Akka and Spark, and tools, like sbt and mil,l but it also makes some room for new usage of Scala, like [Audio and Music](https://index.scala-lang.org/awesome#images-audio-and-video) or [Frontend Development](https://index.scala-lang.org/awesome/web-frontend?sort=stars), in order to promote some new libraries. That makes a lot of things to display in a single page which is not ideal in terms of readability. However I think it is a good page for the curious people to explore and find some interesting libraries to play with.

Where I agree with you is that it would be very useful to have some short version of it, the "awesome-of-the-awesome". I think we should have that in the front page of Scaladex. In terms of implementation it could be, get the 20 most popular libraries, and group them by category (big data, build tools, functionnal programming...).

**Reply by Ichoran** [reply]
Yes, that sounds good!  Top 20 overall would be great.  Probably more useful than 'alphabetized by category'.
